index.js

```diff
@@ -0,0 +1,11 @@
+const fetch = require('node-fetch');
+
+module.exports.register = (options, dependencies, next) => {
+  next();
+};
+
+module.exports.execute = (context) => {
+  const ip = context.requestHeaders['x-forwarded-for'] || context.requestHeaders.remoteAddress;
```

Comment:
what do you think about this strategy @matteofigus ? will this cover most of the cases?